src/components/chart/_macro.njk

```diff
@@ -48,17 +49,23 @@
             {% endif %}
         {% endif %}
     >
-        <figure class="ons-chart">
+        <figure class="ons-chart" aria-describedby="{{ audioDescriptionId }}">
             {{ openingTitleTag | safe }} class="ons-chart__title">{{ params.title }}{{ closingTitleTag | safe }}
             {% if params.subtitle %}
                 {{ openingSubtitleTag | safe }}
                 class="ons-chart__subtitle">{{ params.subtitle }}{{ closingSubtitleTag | safe }}
             {% endif %}
-            {% if params.description %}
-                <p class="ons-u-vh">{{ params.description }}</p>
-            {% endif %}
+
+            <p class="ons-u-vh" id="{{ audioDescriptionId }}">{{ params.description }}</p>
+
             {% if params.iframeUrl %}
-                <div data-chart-iframe id="{{ params.id }}" data-url="{{ params.iframeUrl }}" data-title="{{ params.title }}">
+                <div
+                    data-chart-iframe
```

Comment:
Same here, the pattern we use is to use a class, so this would be something like "ons-chart__iframe"

Reply:
This one is being used as a hook for the pym.js resizing - is a class really a good idea for that, given it might be changed for styling reasons? I could use an id if that is preferred.

Reply:
Its just the pattern we have gone with in the DS, we have done it in other components and I think it helps describe the structure of the elements. It is something we could review but I wouldn't see it changing. For this PR Id rather conform here to the way other components are written and then review that as part of another piece of work on the whole of the DS